buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -3061,32 +3061,13 @@ class BeamModulePlugin implements Plugin<Project> {
       }
       project.ext.toxTask = { name, tox_env, posargs='' ->
         project.tasks.register(name) {
-          dependsOn setupVirtualenv
-          dependsOn ':sdks:python:sdist'
-          if (project.hasProperty('useWheelDistribution')) {
```

Comment:
this was added in #28385 and there was some discussion there. Do they still apply?